[2021-10-29 18:51:11,848] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: example_bash.task_example_id_2 scheduled__2021-10-29T18:34:00+00:00 [queued]>
[2021-10-29 18:51:11,867] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: example_bash.task_example_id_2 scheduled__2021-10-29T18:34:00+00:00 [queued]>
[2021-10-29 18:51:11,868] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-10-29 18:51:11,868] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2021-10-29 18:51:11,868] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-10-29 18:51:11,884] {taskinstance.py:1262} INFO - Executing <Task(BashOperator): task_example_id_2> on 2021-10-29 18:34:00+00:00
[2021-10-29 18:51:11,891] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'example_bash', 'task_example_id_2', 'scheduled__2021-10-29T18:34:00+00:00', '--job-id', '1309', '--raw', '--subdir', 'DAGS_FOLDER/example_bash.py', '--cfg-path', '/tmp/tmp8m01zryd', '--error-file', '/tmp/tmphdvrqynh']
[2021-10-29 18:51:11,892] {standard_task_runner.py:77} INFO - Job 1309: Subtask task_example_id_2
[2021-10-29 18:51:11,889] {standard_task_runner.py:52} INFO - Started process 4279 to run task
[2021-10-29 18:51:11,936] {logging_mixin.py:109} INFO - Running <TaskInstance: example_bash.task_example_id_2 scheduled__2021-10-29T18:34:00+00:00 [running]> on host 85d7243ef68d
[2021-10-29 18:51:11,990] {taskinstance.py:1414} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=example_bash
AIRFLOW_CTX_TASK_ID=task_example_id_2
AIRFLOW_CTX_EXECUTION_DATE=2021-10-29T18:34:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-29T18:34:00+00:00
[2021-10-29 18:51:11,990] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2021-10-29 18:51:11,991] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo "This is a bash test bro 2"']
[2021-10-29 18:51:12,001] {subprocess.py:85} INFO - Output:
[2021-10-29 18:51:12,003] {subprocess.py:89} INFO - This is a bash test bro 2
[2021-10-29 18:51:12,003] {subprocess.py:93} INFO - Command exited with return code 0
[2021-10-29 18:51:12,083] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=example_bash, task_id=task_example_id_2, execution_date=20211029T183400, start_date=20211029T185111, end_date=20211029T185112
[2021-10-29 18:51:12,108] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-10-29 18:51:12,146] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
